packages/nuxt/src/app/nuxt.ts

```diff
@@ -244,13 +244,17 @@ export type ObjectPluginInput<Injections extends Record<string, unknown> = Recor
 export interface CreateOptions {
   vueApp: NuxtApp['vueApp']
   ssrContext?: NuxtApp['ssrContext']
+  /**
+   * The name of the Nuxt application, overrides the default name specified in the Nuxt config (default: `nuxt-app`).
+   */
+  appId?: NuxtApp['_name']
```

Comment:
Would it make sense for this to be `name` instead?

Reply:
I was unsure about this as well. In the nuxt config it is appId, but name would be a natural choice as well. Do you have a preference?

Reply:
I agree, this makes sense! 
However, there was already an `_id` field in the `NuxtApp`, which was used to generate app-wide unique ids in `useId`. I've renamed that field to `_genId`.